BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsClient.java

```diff
@@ -92,15 +92,15 @@ static OneTimeWorkRequest createAnalyticsWorkerRequest(Configuration configurati
                 .build();
     }
 
-    void uploadAnalytics(Context context, Configuration configuration) throws Exception {
+    void uploadAnalytics(Context context, Configuration configuration, BraintreeSharedPreferences braintreeSharedPreferences) throws Exception {
```

Comment:
See PR description for why this changed to non-static, injected param. Open to other ideas of how to handle this given testing framework limitations